### PR TITLE
Fixed the bug where the server cannot unpack a path correctly

### PR DESCRIPTION
**Pull request by zhenlineo**
Added a helper class to unpack/dechunk messages in hex strings to original message

### CODE REVIEW

File: community/bolt/src/test/java/org/neo4j/bolt/v1/packstream/DumpPackStreamV1Message.java [hunk 1]
```diff
@@ -0,0 +1,335 @@
+package org.neo4j.bolt.v1.packstream;
+
```
Comment (chrisvest) [inline community/bolt/src/test/java/org/neo4j/bolt/v1/packstream/DumpPackStreamV1Message.java:21]: This file is missing license headers.

File: community/bolt/src/test/java/org/neo4j/bolt/v1/packstream/DumpPackStreamV1MessageTest.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
+package org.neo4j.bolt.v1.packstream;
+
+import org.junit.Rule;
```
Comment (chrisvest) [inline community/bolt/src/test/java/org/neo4j/bolt/v1/packstream/DumpPackStreamV1MessageTest.java:22]: This file is missing license headers.

File: community/bolt/src/test/java/org/neo4j/bolt/v1/packstream/DumpPackStreamV1MessageTest.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        String input = "00 0F B1 01 8C 4D 79 43    6C 69 65 6E 74 2F 31 2E    30 00 00";
+        input = input.replaceAll( "\\s+", "" );
+        input += input; // same message twice
+        byte[] bytes = DumpPackStreamV1Message.hexStringToBytes( input );
+
+        // When
+        List<Message> messages = DumpPackStreamV1Message.dechunk( bytes );
+
+        // Then
+        assertThat( messages.toString(),
```
Comment (zhenlineo) [inline community/bolt/src/test/java/org/neo4j/bolt/v1/packstream/DumpPackStreamV1MessageTest.java:52]: try
`00  08 B1 01 8C 4D 79 43 6C 69    00 07   65 6E 74 2F 31 2E 30  00 00`

**Comment by jakewins** [comment]
@zhenlineo what's the status of this?

**Comment by jakewins** [comment]
I'm actually going to close this, in favor of https://github.com/neo4j/neo4j/pull/6177 where we reject paths as input altogether. Shout if you disagree!

**Pull request closed without merging**